scan/src/subscriptions/TxSub.re

```diff
@@ -298,19 +298,19 @@ module Msg = {
       identity: string,
       website: string,
       details: string,
-      commissionRate: float,
+      commissionRate: option(float),
       sender: Address.t,
-      minSelfDelegation: float,
+      minSelfDelegation: option(float),
     };
     let decode = json =>
       JsonUtils.Decode.{
         moniker: json |> field("moniker", string),
         identity: json |> field("identity", string),
         website: json |> field("website", string),
         details: json |> field("details", string),
-        commissionRate: json |> field("commission_rate", floatstr),
+        commissionRate: json |> optional(field("commission_rate", floatstr)),
         sender: json |> field("address", string) |> Address.fromBech32,
-        minSelfDelegation: json |> field("min_self_delegation", floatstr),
+        minSelfDelegation: json |> optional(field("min_self_delegation", floatstr)),
```

Comment:
This is not actually getting used?

Reply:
My bad, I forgot to make PR of this fix.